assets/components/contributionPaymentCtas/contributionPaymentCtas.jsx

```diff
@@ -116,6 +116,7 @@ function OneOffCta(props: {
       accessibilityHint={`proceed to make your ${spokenType} contribution`}
       url={clickUrl}
       onClick={onCtaClick(props.isDisabled, props.resetError)}
+      id="qa-contribute-button"
```

Comment:
Would it be safer if the ids for one-off and regular were different?

Reply:
They'll never be on the page at the same time, but I can change them if you think it's best?

Reply:
No need to worry if they can't appear together 👍

Reply:
> They'll never be on the page at the same time
Heh. Let's see. 🙈

Reply:
:trollface: